lib/Runtime/Library/JSONStringBuilder.h

```diff
@@ -0,0 +1,42 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+#pragma once
+
+namespace Js
+{
+
+class JSONStringBuilder
+{
+private:
+    ScriptContext* scriptContext;
```

Comment:
Nit: all of these fields are consts? like char16* const buffer etc?

Reply:
i'm filling the buffer, so that one isn't really a const. i can change some of these to be const, but tbh i find const members to be more effort then they are worth

---
In reply to: [147309108](https://github.com/Microsoft/ChakraCore/pull/4077#discussion_r147309108) [](ancestors = 147309108)